### PR TITLE
Removing ambiguity in the Template Override ModelAdmin documentation

### PR DESCRIPTION
**Pull request by JoeWHoward**
As it stands, the docs on this topic (Overriding Templates in ModelAdmin) seem to indicate that the /modeladmin/ directory should be placed in the project root folder.  I think a rewording of line 144 is in order as well, but I don't know a good way to phrase that.

### CODE REVIEW

**Comment by jacobtoppm** [comment]
Thanks for contributing! I definitely think this clears up some ambiguity. For line 144, what about 'project's apps' - do you think that's clearer? I agree that bit still sounds a little like it's referring to the root.

**Comment by thibaudcolas** [comment]
👋  does this replace #5083 or is that other PR still relevant?

File: docs/reference/contrib/modeladmin/primer.rst [hunk 1]
```diff
@@ -143,14 +143,14 @@ Overriding templates
 For all modeladmin views, Wagtail looks for templates in the following folders
 within your project, before resorting to the defaults:
```
Comment (JoeWHoward) [inline docs/reference/contrib/modeladmin/primer.rst:144]: within your project or app, before resorting to the defaults:

**Review by JoeWHoward — COMMENTED** [review]
From the discussion in #5083 it seems that the templates/modeladmin folder can exist in either the project directory or an app directory, but the inclusion of the templates parent folder is the same either way.

I understand that this is something that should be "inferred" from working with Django, but I think the example paths are misleading if no mention of the templates directory is made on the page.

**Comment by lb-** [comment]
@JoeWHoward  I made that last little change locally, thanks so much for taking the time to improve the docs.

This has now been merged in via 4ab7080fb8e3def96ec6e82df20c9b38e1a93f2e

**Pull request closed without merging**